man/man5/zfs-module-parameters.5

```diff
@@ -1765,6 +1765,20 @@ table.
 Default value: \fB131,072\fR.
 .RE
 
+.sp
+.ne 2
+.na
+\fBfletcher_4_impl_set\fR (string)
+.ad
+.RS 12n
+Select a fletcher 4 implementation to use.
+Two selectors are supported: \fBfastest\fR and \fBoriginal\fR. If there exists
```

Comment:
Primarily I want to ensure the names used in this patch match those used in the raidz patch as much as possible.  Users appreciate consistency.  Let's go with scalar.